mirascope/core/base/messages.py

```diff
@@ -2,26 +2,45 @@
 
 from collections.abc import Sequence
 from typing import Any
+from wave import Wave_read
 
 from ._utils._convert_messages_to_message_params import (
     Image,
     convert_message_content_to_message_param_content,
 )
 from .message_param import AudioPart, BaseMessageParam, ImagePart, TextPart
+from .types import AudioSegment
 
 
 class Messages:
     Type = (
         str
-        | Sequence[str | Image.Image | TextPart | ImagePart | AudioPart]
+        | Sequence[
+            str
+            | Image.Image
```

Comment:
nit: same there and and below, colocate `Image.Image` by putting it right after `ImagePart`